### PR TITLE
[Hardware][CPU] Just warn on missing logits soft cap implementation.

### PR DESCRIPTION
**Pull request by janimo**
Turn an error into a warning and allow Gemma2 and PaliGemma 2 to run on the CPU backend with seemingly normal outputs.

From https://huggingface.co/blog/gemma2

"The Gemma 2 team observed very minor differences when soft-capping is removed during inference."

### CODE REVIEW

**Comment by janimo** [comment]
close in favour of #11169
